src/Catalog/Monitoring/Validation/Test/EndpointToTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test
+{
+    /// <summary>
+    /// Represents an endpoint to run tests against.
+    /// </summary>
+    public interface IEndpointToTest
+    {
+    }
+
+    /// <summary>
+    /// Represents the registration blobs endpoint, which stores metadata about packages.
+    /// </summary>
+    public class RegistrationEndpoint : IEndpointToTest
```

Comment:
>RegistrationEndpoint [](start = 17, length = 20)

Why is this a class? There is no interface. These look like a candidate for enum